easybuild/easyconfigs/s/strace/strace-6.14-GCCcore-14.3.0.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'strace'
+version = '6.14'
+
+homepage = 'https://strace.io/'
+description = """
+strace is a diagnostic, debugging and instructional userspace utility for Linux. It is used to monitor and tamper with
+interactions between processes and the Linux kernel, which include system calls, signal deliveries, and changes of
+process state.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '14.3.0'}
+
+source_urls = ['https://github.com/strace/strace/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_XZ]
+checksums = ['244f3b5c20a32854ca9b7ca7a3ee091dd3d4bd20933a171ecee8db486c77d3c9']
+
+builddependencies = [
+    ('binutils', '2.44'),
+    ('libunwind', '1.8.2'),
+]
```

```suggestion
builddependencies = [
    ('binutils', '2.44'),
]

dependencies = [
    ('libunwind', '1.8.2'),
]
```

To address build failure seen in one test report.